config.tf

```diff
@@ -1,14 +1,50 @@
+locals {
+  vnet_rg   = "aks-infra-${var.env}-rg"
+  vnet_name = "core-${var.env}-vnet"
+
+  criticality = {
+    sbox     = "Low"
+    aat      = "High"
+    prod     = "High"
+    ithc     = "Medium"
+    ldata    = "Medium"
+    perftest = "Medium"
+    demo     = "Medium"
+  }
+
+  env_display_names = {
+    sbox     = "Sandbox"
+    aat      = "Staging"
+    prod     = "Production"
+    ithc     = "ITHC"
+    ldata    = "LDATA"
+    perftest = "Test"
+    demo     = "Demo"
+  }
+
+  tags = {
+    "managedBy"          = "Platform Engineering"
+    "solutionOwner"      = "CFT"
+    "activityName"       = "AKS"
+    "dataClassification" = "Internal"
+    "automation"         = ""
+    "costCentre"         = "10245117" // until we get a better one, this is the generic cft contingency one
+    "environment"        = local.env_display_names[var.env]
+    "criticality"        = local.criticality[var.env]
+  }
+}
+
```

Comment:
This was moved out as it will be different for each environment

Reply:
Moved into CFT component